src/System.Diagnostics.Debug/tests/DebugTests.cs

```diff
@@ -9,6 +9,7 @@ namespace System.Diagnostics.Tests
     public class DebugTests
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Net46)]
         public void Asserts()
         {
             VerifyLogged(() => { Debug.Assert(true); }, "");
```

Comment:
Do you know why Debug.Assert doesn't work the same? Is it because we are running on a release build of the framework? 

If this is a netfx issue you should probably mark this as TargetFrameworkMonikers.NetFramework